packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4540,6 +4548,30 @@ class Semantics extends SingleChildRenderObjectWidget {
     return textDirection ?? (label != null || value != null || hint != null ? Directionality.of(context) : null);
   }
 
+  /// The handler for [SemanticsAction.tap].
```

Comment:
copy all the documentation you add for the callbacks above into these too, they're even more the front-line docs.

Reply:
Done.